airgun/browser.py

```diff
@@ -420,7 +420,7 @@ def check_alert(self, locator):
             self.logger.info(f'No {locator} type alert detected !')
         return False
 
-    def get_alert(self):
+    def get_alert(self, squash=False):
```

Comment:
Can you add a docstring for squash ?

Does the `squash` word a satellite/airgun or its dependencies terminology ?